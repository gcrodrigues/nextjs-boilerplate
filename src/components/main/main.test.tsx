import { render, screen } from '@testing-library/react'

import Main from '.'

describe('<Main />', () => {
  it('should render the heading', () => {
    const { container } = render(<Main />)

    expect(
      screen.getByRole('heading', { name: /Aug\/2021/i })
    ).toBeInTheDocument()

    expect(container).toMatchSnapshot()
  })
})
