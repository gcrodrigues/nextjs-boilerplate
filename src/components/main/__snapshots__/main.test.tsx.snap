// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render the heading 1`] = `
.c0 {
  background-color: #06092b;
  color: #fff;
  width: 100%;
  height: 100%;
  text-align: center;
  padding: 3rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c1 {
  font-size: 3rem;
  margin-bottom: 2rem;
}

.c2 {
  font-size: 2rem;
  font-weight: 400;
}

<div>
  <main
    class="c0"
  >
    <h1
      class="c1"
    >
      Boilerplate Aug/2021
    </h1>
    <h2
      class="c2"
    >
      Created with Typescript, ReactJS, NextJS, Styled Components, Jest, Husky, Eslint and Prettier
    </h2>
  </main>
</div>
`
